src/app/zap-templates/templates/app/helper.js

```diff
@@ -195,8 +195,10 @@ function asChipUnderlyingType(label, type)
 // These helpers only works within the endpoint_config iterator
 
 // List of all cluster with generated functions
-var endpointClusterWithInit =
-    [ 'Basic', 'Identify', 'Groups', 'Scenes', 'Occupancy Sensing', 'On/off', 'Level Control', 'Color Control', 'IAS Zone' ];
+var endpointClusterWithInit = [
+  'Basic', 'Air Pressure Measurement', 'Identify', 'Groups', 'Scenes', 'Occupancy Sensing', 'On/off', 'Level Control',
```

Comment:
Please don't add this cluster here if it does not in fact need an init function.  Better to remove the empty function and not add it to this list.